src/ngraph/ops/slice.cc

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+* Copyright 2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "ops/slice.h"
+#include "../operator/tensor/matrix_op-inl.h"
+#include "ngraph_sgcompiler_utils.h"
+
+namespace ngraph_bridge {
+
+NgraphNodePtr create_slice_op(const NgraphNodePtr& node,
+                              const nnvm::NodeAttrs& attrs) {
+  const mxnet::op::SliceParam& param =
+      nnvm::get<mxnet::op::SliceParam>(attrs.parsed);
+  nnvm::TShape tshape = NShape_to_TShape(node->get_shape());
+  ngraph::Coordinate ng_begin, ng_end, ng_step;
+  ngraph::AxisSet axes;
```

Comment:
It would be nice for `axes` to be named that made its purpose / intent clearer to casual readers of the code.